### ISSUE TITLE
Incorrect VAT rates for Estonia? (EE)

### ISSUE THREAD

**Issue by mtarnovan** [ibericode/vat-rates#22]
According to [this](https://marosavat.com/estonia-increase-vat-rate-2025/#:~:text=Furthermore%2C%20the%20Estonian%20government%20has,to%2024%25%20by%20July%202025) Estonia has two reduced VAT rates applicable in 2024 of 9% and 5%, this repo only has the 9% one.

Additionally, according to the same article, the increase in standard VAT rate from 20% to 22% is effective from 2025-06-01, not 2024-01-01 as it shows currently in the JSON file.

However, in [the official docs](https://www.emta.ee/en/business-client/taxes-and-payment/value-added-tax#from-01012025) I can't find a reference to the new 22% rate.

**Comment by dannyvankooten** [comment]
@mtarnovan [This source](https://www.emta.ee/en/business-client/taxes-and-payment/value-added-tax/vat-rates-and-supply-exempt-tax#standard) references the 22% standard VAT rate effective as per 2024-01-01. 

[Your link](https://marosavat.com/estonia-increase-vat-rate-2025/#:~:text=Furthermore%2C%20the%20Estonian%20government%20has,to%2024%25%20by%20July%202025) talks about an additional increase from 22% to 24% by July 2025 (not 20 to 22%). 

As for the reduced rates, I am thinking of removing those from this repository altogether as it's a bit of a hassle to come up with a generalized way that works across all EU countries... Plus as you said, they are not 100% correct right now anyway.

**Comment by mtarnovan** [comment]
Thanks for the quick reply.

> As for the reduced rates, I am thinking of removing those from this repository altogether as it's a bit of a hassle to come up with a generalized way that works across all EU countries...

It is indeed annoying that there is no generalized way, I wanted to send a PR to add the 5% one, but wasn't even sure, would this be `reduced1/reduced2` or `reduced/super_reduced` or something else...? However, please reconsider removing them, we really need those. Even if they are not 100% correct all the time, it's still useful to have a change signal from this repo.

**Comment by KarelBrijs** [comment]
@dannyvankooten I agree with @mtarnovan regarding removed reduces rates even though it's not always 100% correct, this is currently the best (and I think only) library out there. Hats off for maintaining it!

**Comment by ragulka** [comment]
I didn't notice this issue before I already opened a PR to add the 2nd reduced rate in #25.

**Comment by BenHarkins** [comment]
Is there a way to separate out the change for the increase in the top rate (to 24%) from the change for support for a second reduced rate?
